package.json

```diff
@@ -43,10 +43,11 @@
   "dependencies": {
     "@ant-design/icons": "^4.5.0",
     "@types/react-datepicker": "^3.1.8",
-    "antd": "^4.13.0",
+    "antd": "^4.16.2",
     "date-fns": "^2.21.1",
     "lodash": "^4.17.21",
-    "react-datepicker": "^4.1.1"
+    "react-datepicker": "^4.1.1",
+    "styled-css-grid": "1.2.1"
```

Comment:
Let's omit this dependency and just use CSS grid directly in this library. The library has not been updated since 2019 and depends on an outdated version of `styled-components`.